frontend/src/components/my-items/AddItemForm.jsx

```diff
@@ -78,7 +103,14 @@ export default function AddItemForm (props) {
         </Form.Group>
         <Button variant="outline-primary" type="submit" className="me-1 button"
           onClick={handleSubmit}>Submit</Button>
-        <Button variant="outline-danger" type="reset" className="button">Reset</Button>
+        <Button variant="outline-danger" type="reset" className="button" onClick={handleReset}>Reset</Button>
+        {showAlert
+          ? (
+            <Alert variant="warning" onClose={() => setShowAlert(false)} dismissible>
```

Comment:
The other validation alerts on the site use the 'danger' variant. Don't feel strongly either way but we should be consistent.

Reply:
Changing them to warning since danger felt a little aggressive on the modals